### PR TITLE
Feature/image updates

### PR DESCRIPTION
**Pull request by imranhaidercogworks**
**Updated the image builder to be able to handle lazy load option and default height/width**:

---

**Checklist**:

- [ ] My PR has a descriptive title (not a vague title like `Needed this`)
- [ ] Title and all the commits follow [commitlint guidelines](https://github.com/conventional-changelog/commitlint#what-is-commitlint)
- [ ] My PR targets the `develop` branch or appropriate `release` branch
- [ ] I've placed the link to the corresponding Trello card or issue which this PR fixes/address below ⤵️

---

Fixes: ### ISSUE_OR_TRELLO_LINK ###

### CODE REVIEW

File: Source/Cogworks.Umbraco.Essentials/Builders/Interfaces/IResponsiveImageBuilder.cs [hunk 1]
```diff
@@ -19,7 +19,9 @@ public interface IResponsiveImageBuilder
             int? height = null,
             bool includeRetina = true,
             bool enableWebP = false,
-            int? quality = null);
+            int? quality = null,
+            bool isLazy = true,
+            bool addDefaultDimensions = false);
```
Comment (aochmann) [inline Source/Cogworks.Umbraco.Essentials/Builders/Interfaces/IResponsiveImageBuilder.cs:24]: I would rename it to `defaultDimensions` or `enableDefaultDimensions` 🤔 What do you think ?

File: Source/Cogworks.Umbraco.Essentials/Builders/Interfaces/IResponsiveImageBuilder.cs [hunk 2]
```diff
@@ -19,7 +19,9 @@ public interface IResponsiveImageBuilder
             int? height = null,
             bool includeRetina = true,
             bool enableWebP = false,
-            int? quality = null);
+            int? quality = null,
+            bool isLazy = true,
```
Comment (aochmann) [inline Source/Cogworks.Umbraco.Essentials/Builders/Interfaces/IResponsiveImageBuilder.cs:23]: IMO we should disable it by default and only enabling if we want to use it.

File: Source/Cogworks.Umbraco.Essentials/Models/ResponsiveImage.cs [hunk 3]
```diff
@@ -11,5 +11,11 @@ public class ResponsiveImage
         public string ContainerClass { get; set; }
 
         public string AltText { get; set; }
+
+        public string LazyLoadValue { get; set; }
```
Comment (aochmann) [inline Source/Cogworks.Umbraco.Essentials/Models/ResponsiveImage.cs:15]: Should we rename it to `Loading` or `LoadingSetting` or `LoadingValue` or `LazyLoading` or `[Lazy]LoadingConfiguration`  like on https://github.com/thecogworks/Cogworks.Umbraco.Essentials/blob/168a064a092de44900bbc276a6276dffa6913ed4/Source/Cogworks.Umbraco.Essentials/ViewFiles/_ResponsiveImage.cshtml#L24

**Review by aochmann — APPROVED** [review]

File: Source/Cogworks.Umbraco.Essentials/Builders/Interfaces/IResponsiveImageBuilder.cs (hunk 1 above)
Comment (trt) [reply Source/Cogworks.Umbraco.Essentials/Builders/Interfaces/IResponsiveImageBuilder.cs:24]: Agree -> addDefaultDimension sounds like a function.

**Review by trt — APPROVED** [review]

**Review by aochmann — APPROVED** [review]

**Pull request merged**